docs/Plugins/bitbucket.md

```diff
@@ -12,6 +12,9 @@ This plugin collects various entities from Bitbucket, including pull requests, i
 
 As of v0.14.2, `bitbucket` plugin can only be invoked through DevLake API. Its support in Config-UI is WIP.
 
+## Supported Versions
+Available for BitBucket Cloud
```

Comment:
1. Can you modify this line to 
```
"Available for BitBucket Cloud. Check [this doc](relative url of the supported data sources) for more details."
```

2. Add the URL to every plugin's supported versions.

On a side note, we can put the detailed versions here, but remember that we have to modify two docs once a new version has been supported for a plugin.